apps/dataflow/src/scraper/esim/osm_building_to_json.py

```
@@ -0,0 +1,607 @@
+                "role": m.attrib.get("role", ""),
+            }
+            for m in r.findall("member")
+        ]
+        relations.append({"id": rid, "tags": tags, "members": members})
+
+    entrance_nodes.update(
+        nid for nid, t in node_tags.items() if "entrance" in t or "door" in t
+    )
+
+
+def _collect_buildings() -> dict[str, dict[str, Any]]:
```

Ruff (0.14.14)

[warning] Too many branches (13 > 12)

(PLR0912)